src/grammar.lalrpop

```diff
@@ -162,6 +162,7 @@ UOp: UnaryOp<RichTerm> = {
     "isStr" => UnaryOp::IsStr(),
     "isFun" => UnaryOp::IsFun(),
     "isList" => UnaryOp::IsList(),
+    "isRec" => UnaryOp::IsRec(),
```

Comment:
Please, please, please, pretty please: name it `"isRecord"`. I keep reading “is recursive” :slightly_smiling_face: .